homeassistant/components/bluecurrent/sensor.py

```diff
@@ -0,0 +1,73 @@
+"""Support for BlueCurrent sensors."""
+from __future__ import annotations
+
+from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import BlueCurrentEntity, Connector
+from .const import DOMAIN, GRID_SENSORS, SENSORS, TIMESTAMP_KEYS
+
+PARALLEL_UPDATES = 1
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up Blue Current sensors."""
+    connector: Connector = hass.data[DOMAIN][entry.entry_id]
+    sensor_list: list[BlueCurrentSensor] = []
+    for evse_id in connector.charge_points.keys():
```

Comment:
You iterate over keys when you iterate a dict anyway, so
```suggestion
    for evse_id in connector.charge_points:
```